packages/pocl-job/src/io/s3.js

```diff
@@ -47,13 +44,6 @@ export const refreshS3Metadata = async () => {
     const filename = file.Key.split('/').pop()
 
     console.log(`Processing ${filename}`)
-
-    const dynamicsRecord = await salesApi.getTransactionFile(filename)
-    if (!dynamicsRecord || !DYNAMICS_IMPORT_STAGE.isAlreadyProcessed(dynamicsRecord.status.description)) {
-      await storeS3Metadata(file.ETag, filesize(file.Size), filename, file.Key, moment(new Date(file.LastModified)))
-    } else {
-      console.log(`${file.Key} is already processed, skipping`)
-    }
```

Comment:
Not sure if we want to delete this and the `storeS3Metadata` function – I think this can still be used for other file uploads as well and we don't want to blow up the direct debit process. However if we do keep it then it might make sense to move that function out of a file called ftp-to-s3.js anyway.